src/sentry/api/bases/sentryapps.py

```diff
@@ -99,6 +103,45 @@ def dispatch(self, request, *args, **kwargs):
 class SentryAppsBaseEndpoint(IntegrationPlatformEndpoint):
     permission_classes = (SentryAppsPermission,)
 
+    def _get_organization_slug(self, request):
+        organization_slug = request.json_body.get("organization")
+        if not organization_slug or not isinstance(organization_slug, string_types):
+            error_message = """
+                Please provide a valid value for the 'organization' field.
+            """
+            raise ValidationError({"organization": to_single_line_str(error_message)})
+        return organization_slug
+
+    def _get_organization_for_superuser(self, organization_slug):
+        try:
+            return Organization.objects.get(slug=organization_slug)
+        except Organization.DoesNotExist:
+            error_message = """
```

Comment:
```suggestion
            error_message = u"""
```

Reply:
I realize now that `to_single_line_str` handles unicode but with the conversion to Py3 happening, it's better to just have everything as unicode.

Reply:
Will do!

Reply:
This is a good change regardless of `to_single_line_str` since the `format` call would blowup if `organization_slug` was unicode.

Reply:
Done!